engine/src/flutter/lib/web_ui/skwasm/canvas_text.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_LIB_WEB_UI_SKWASM_CANVAS_TEXT_H_
+#define FLUTTER_LIB_WEB_UI_SKWASM_CANVAS_TEXT_H_
+
+#include "flutter/display_list/dl_text.h"
+#include "third_party/skia/include/core/SkRefCnt.h"
+
+#include <memory>
+
+namespace flutter {
+extern std::shared_ptr<DlText> textFromBlob(const sk_sp<SkTextBlob>& blob);
```

Comment:
`extern` is redundant on a function declaration as long as all users and implementors include this header.  You may have to pull this header into its own module to make that easy.